include/singa/utils/context.h

```diff
@@ -23,6 +23,7 @@
 #define SINGA_UTILS_CONTEXT_H_
```

Comment:
I think without device mappings, the Context class cannot support the math blob operations. Therefore, there would be errors from compilation.